src/views/info.ts

```diff
@@ -15,20 +15,20 @@ import { showAlertDialog } from "@ha/dialogs/generic/show-dialog-box";
 import { HomeAssistant } from "@ha/types";
 
 import { getKnxInfo, processProjectFile, removeProjectFile } from "../services/websocket.service";
-import { KNXInfo, KNXProjectInfo } from "../types/websocket";
+import { KNXInfoData, KNXProjectInfo } from "../types/websocket";
 import { localize } from "../localize/localize";
 import { KNXLogger } from "../tools/knx-logger";
 import { VERSION } from "../version";
 
-const logger = new KNXLogger("overview");
+const logger = new KNXLogger("info");
 
-@customElement("knx-overview")
-export class KNXOverview extends LitElement {
+@customElement("knx-info")
+export class KNXInfo extends LitElement {
   @property({ type: Object }) public hass!: HomeAssistant;
 
   @property({ type: Boolean, reflect: true }) public narrow!: boolean;
 
-  @state() private knxInfo: KNXInfo | null = null;
+  @state() private knxInfo: KNXInfoData | null = null;
```

Comment:
```suggestion
  @state() private knxInfoData: KNXInfoData | null = null;
```

Maybe rename this then too to reflect the type.